regulations/views/comment.py

```diff
@@ -23,6 +31,7 @@ def upload_proxy(request):
     url = s3.generate_presigned_url(
         ClientMethod='put_object',
         Params={
+            'ContentLength': size,
```

Comment:
This is interesting. What happens if the uploaded file is actually larger than the content length provided? Will S3 trim the file? That'd be a pretty nice security mechanism.

Related, are there any special values we need to keep in mind? Do 0 or -1 mean "arbitrary size", for example?